backend/apps/slack/management/commands/owasp_match_channels.py

```
@@ -0,0 +1,36 @@
+from apps.owasp.models.chapter import Chapter
+from apps.owasp.models.committee import Committee
+from apps.owasp.models.project import Project
+
+class Command(BaseCommand):
+    help = 'Populate EntityChannel records for Chapters, Committees, and Projects based on Slack data.'
+
+    def handle(self, *args, **options):
+        created = 0
+        # Example: match by name, can be improved
+        for model, app_label, model_name in [
+            (Chapter, 'owasp', 'chapter'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)